app/scss/_goalTile.scss

```diff
@@ -0,0 +1,54 @@
+@import 'general';
+@import 'colors';
+
+.goalTile {
+  position: relative;
+  border-radius: 6px;
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  width: 90%;
+  max-width: 400px;
+  height: 98px;
+  z-index: 0;
+  cursor: pointer;
+}
+
+.goalTile_avatarContainer {
+  height: 83px;
+  width: 68px;
+  margin: 5px;
+
+  img {
+    max-height: 84px;
+  }
+}
+
+.goalTile_nameContainer {
+  position: absolute;
+  right: 10px;
+  width: 75%;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  top: 20px;
+}
+
+.goalTile_rating {
+  position: absolute;
+  right: 10px;
+  top: 50px;
+  width: 31px;
+  height: 31px;
```

Comment:
Can you check though this css and check if some of these numbers would be better off in vars